app/code/Magento/ReviewGraphQl/etc/schema.graphqls

```diff
@@ -0,0 +1,89 @@
+# Copyright © Magento, Inc. All rights reserved.
+# See COPYING.txt for license details.
+
+type Query {
+    productReviews (
+        sku: String! @doc(description: "The product SKU to get reviews for"),
+        pageSize: Int = 20 @doc(description: "Specifies the maximum number of results to return at once. This attribute is optional."),
+        currentPage: Int = 1 @doc(description: "Specifies which page of results to return. The default value is 1."),
+        sort: ReviewSortInput @doc(description: "Specifies which field to sort on, and whether to return the results in ascending or descending order.")
+    ): ProductReviews @resolver(class: "Magento\\ReviewGraphQl\\Model\\Resolver\\Product\\Reviews") @doc(description: "The product review query returns information about approved product reviews")
+    customerProductReviews (
+        pageSize: Int = 20 @doc(description: "Specifies the maximum number of results to return at once. This attribute is optional."),
```

Comment:
To accommodate detailed review view page, let's introduce optional `filter` which will have `review_id` as the first available filter option.